nemo_curator/utils/file_utils.py

```diff
@@ -182,8 +182,7 @@ def _save_jsonl(documents, output_path, start_index=0, max_index=10000, prefix=N
     """Worker function to write out the data to jsonl files"""
 
     def _output_json(document):
-        myjson = json.dumps(document, ensure_ascii=False)
-        return myjson.encode("utf-8")
+        return document.strip().encode("utf-8")
```

Comment:
I would prefer renaming this function since it doesn't output a json string anymore, it assumes that the input is already a json. Maybe something like `_encode_text` or something similar to explain what it does.

Reply:
Sure! It makes sense! I will change it now.

Reply:
Done!